### ISSUE TITLE
Support re-export of derive_builder

### ISSUE THREAD

**Issue by ryo33** [colin-kiegel/rust-derive-builder#274]
## Background

[parol](https://github.com/jsinger67/parol) generates code that has several external crates, including derive_builder. And, [parol_runtime](https://github.com/jsinger67/parol_runtime) re-exports the crates for ease of use.

## Problem

I know `derive_builder` doesn't support crate renaming, but it also seems not to support re-exporting.
So we need a user-confusing workaround in `lib.rs` or `main.rs` as below:

```rust
use parol_runtime::derive_builder;
```

## Solution

Replace `::derive_builder" with "derive_builder" in generated code.

It may break projects in Rust 2015 edition, and we may need a feature toggle for this behavior.

**Comment by TedDriggs** [comment]
I think I'm reading that `parol` is generating structs that are then trying to derive a builder, without requiring the user of `parol` to directly depend on `derive_builder`; does that accurately describe the issue?

**Comment by ryo33** [comment]
Yes, it does.

**Comment by TedDriggs** [comment]
It seems that the common solution for this is to expose a `crate` attribute, e.g.

```rust
#[derive(Builder)]
#[builder(crate = "parol_runtime::derive_builder")]
```

That's how `serde` does it, and that seems like that should not have any backwards compatibility risks. Would `parol_runtime` be able to use such a solution?

**Comment by ryo33** [comment]
Yeah, it seems nice.

**Comment by TedDriggs** [comment]
This is published as part of 0.12.0
